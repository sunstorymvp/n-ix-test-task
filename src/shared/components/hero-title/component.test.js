import renderer from 'react-test-renderer';

import HeroTitle from './component';

test('it renders', () => {
  const tree = renderer.create(<HeroTitle>title</HeroTitle>).toJSON();

  expect(tree).toMatchSnapshot();
});
