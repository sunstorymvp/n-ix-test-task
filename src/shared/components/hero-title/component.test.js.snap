// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<h1
  className="root"
>
  title
</h1>
`;
